src/Nordea/Components/RadioButton.elm

```diff
@@ -65,6 +66,11 @@ import Nordea.Resources.Colors as Colors
 import Nordea.Themes as Themes
 
 
+type Size
+    = DefaultSize
+    | Small
```

Comment:
We have used these terms for this in the other components, should we stick with that?
```suggestion
type Variant
    = Standard
    | Small
```

Reply:
Related to @hhaga 's comment. When we tried following the convention we got conflicting values "Standard" of more than 1 type. We didn't see how to resolve that, other than having different values. And then we thought "should we in general go for expressive / specific names rather than broad / general names that can have overlap and conflict" ?

This might just be our lack of knowledge regarding Elm here :D

Reply:
Ahh, I see! So, instead of adding a new type and field for size, what about adding a `Small` variant to the existing Appearance type? Do you think that would make sense?

Reply:
@hhaga @fredrikloberg tagging both since same discussion

Specifically our issue is:

`appearance: Standard
, variant: Standard
`

We can't solve this with doing 


`appearance: Appearance.Standard
, variant: Variant.Standard
`

Reply:
I suggest to extend the exisitng Appearance type like so:
```elm
type Appearance
    = Standard
    | Simple
    | ListStyle
    | Small
```

Reply:
@fredrikloberg adding it to Appearance makes sense if we never want to combine listStyle + small.
That coupling wasn't something we thought was in the requirements, but it would fix the current issue.
Do you think we should pay that limitation as the price?

Reply:
Yes, I think that makes sense. The ListStyle is a very different styling anyway